pi5/board.go

```diff
@@ -308,8 +191,8 @@ func (b *pinctrlpi5) createGpioPin(mapping gl.GPIOBoardMapping) *gpioPin {
 
 type pinctrlpi5 struct {
 	resource.Named
-	mu            sync.RWMutex
-	convertConfig ConfigConverter
+	resource.TriviallyReconfigurable
+	mu sync.RWMutex
```

Comment:
I should change this from an RWMutex to just a normal Mutex. We don't need the extra complexity of a mutex where multiple goroutines can acquire read-locks simultaneously (which is useful when building a database, and approximately nothing else).